examples/scripts/alignprop.py

```diff
@@ -132,4 +132,4 @@ def image_outputs_logger(image_pair_data, global_step, accelerate_logger):
     # Save and push to hub
     trainer.save_model(training_args.output_dir)
     if training_args.push_to_hub:
-        trainer.push_to_hub()
+        trainer.push_to_hub(dataset_name=script_args.dataset_name)
```

Comment:
Nice, I did not know we could pass this arg in `push_to_hub()`!

Reply:
push_to_hub forwards the kwargs to create_model_card. So if you have this args in create_model_card, you can pass it.